### PR TITLE
fix: add build wes adapter step to run dev script and readme

### PR DESCRIPTION
**Pull request by evanyui**
Issue #, if available:

**Description of Changes**
Fix dev-run.sh by adding the wes_adapter build step
Updated readme and have another option to run local build agc

[//]: #  (A description of the change that you made and the new user experience that it creates)

**Description of how you validated changes**
Ran dev-run.sh, `account activate`, `context deploy`, `run workflow`

[//]: #  (A description of you validated your changes and any relevant logs that show your change works)
```
$ agc account activate
2022-02-04T14:34:56-08:00 𝒊  Activating AGC with bucket '' and VPC ''
Bootstrapping CDK... [--o-] 13s
Activating account... [-o--] 2m30s                                                                              $ agc context deploy myContext
2022-02-04T14:45:33-08:00 𝒊  Deploying context(s)
Deploying resources for context(s) [myContext] [o---] 9m26s                                                     2022-02-04T14:55:01-08:00 𝒊  Successfully deployed context(s) [myContext]
$ agc workflow run hello --context myContext
2022-02-04T14:58:32-08:00 𝒊  Running workflow. Workflow name: 'hello', Arguments: '', Context: 'myContext'
112b0ff1-e22d-4fa9-a4e3-d0ed669a8eaf
```

**Checklist**

- [x] If this change would make any existing documentation invalid, I have included those updates within this PR
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have linted my code before raising the PR
- [x] Title of this Pull Request follows Conventional Commits standard: https://www.conventionalcommits.org/en/v1.0.0/

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by tneely — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -105,19 +106,21 @@ export ECR_CROMWELL_TAG=<some-value>
...
 your Amazon Genomics CLI account region (`agc account activate`). The `./scripts/run-dev.sh` contains logic to determine the current
 dev versions of the images which you would typically use. You may also use production images, the current values of which will
 be written when you activate an account with the production version of Amazon Genomics CLI. If you have customized containers that you 
 want to develop against you can specify these however you will need to make these available if you wish to make pull requests
 with code that depends on them.
 
+#### Option 2. Running with local release
+To run a release version of Amazon Genomics CLI from your local build, first build your changes and then run `make release`.
+This will create a release bundle `dist/` at this package root directory. Run the `install.sh` script to install your 
+release version of Amazon Genomics CLI. After installing, you should be able to run `agc` on the terminal.
```
Comment (waliag87) [inline README.md:122]: Should we add that this will replace any other versions of agc installed, including if they are on a stable version?

File: README.md [hunk 2]
```diff
@@ -105,19 +106,21 @@ export ECR_CROMWELL_TAG=<some-value>
...
 that will be deployed for your contexts. They are written as Systems Manager Parameter Store variables when you activate
 your Amazon Genomics CLI account region (`agc account activate`). The `./scripts/run-dev.sh` contains logic to determine the current
 dev versions of the images which you would typically use. You may also use production images, the current values of which will
 be written when you activate an account with the production version of Amazon Genomics CLI. If you have customized containers that you 
 want to develop against you can specify these however you will need to make these available if you wish to make pull requests
 with code that depends on them.
 
+#### Option 2. Running with local release
+To run a release version of Amazon Genomics CLI from your local build, first build your changes and then run `make release`.
+This will create a release bundle `dist/` at this package root directory. Run the `install.sh` script to install your
```
Comment (waliag87) [inline README.md:121]: I think we have 2-3 different install.sh scripts. We should specify which one

File: README.md [hunk 3]
```diff
@@ -105,19 +106,21 @@ export ECR_CROMWELL_TAG=<some-value>
 export ECR_NEXTFLOW_ACCOUNT_ID=<some-value>
 export ECR_NEXTFLOW_REGION=<some-value>
 export ECR_NEXTFLOW_TAG=<some-value>
-export ECR_WES_ACCOUNT_ID=<some-value>
-export ECR_WES_REGION=<some-value>
-export ECR_WES_TAG=<some-value>
 ```
 
-These environment variables point to the ECR account, region, and tags of the Cromwell engine and WES adaptor containers respectively
+These environment variables point to the ECR account, region, and tags of the Cromwell and Nextflow engine respectively
```
Comment (elliot-smith) [inline README.md:111]: Not really part of the PR but can you please add the MiniWDL values above as well?

**Comment by elliot-smith** [comment]
Can you please test context deploy as well as that was the command that was causing the issue for everyone.

File: README.md (hunk 1 above)
Comment (evanyui) [reply README.md:122]: @waliag87 Added note that this will replace the local version if they install using this option
@elliot-smith Added missing some details, installing using .install.sh from dist/ should replace their global version of AGC

File: README.md (hunk 2 above)
Comment (evanyui) [reply README.md:121]: Done! thanks!

File: README.md (hunk 3 above)
Comment (evanyui) [reply README.md:111]: No problem

**Comment by evanyui** [comment]
> Can you please test context deploy as well as that was the command that was causing the issue for everyone.

Tested with context deploy and run workflow. Updated description

File: README.md [hunk 4]
```diff
@@ -105,19 +106,25 @@ export ECR_CROMWELL_TAG=<some-value>
...
-export ECR_WES_ACCOUNT_ID=<some-value>
-export ECR_WES_REGION=<some-value>
-export ECR_WES_TAG=<some-value>
+export ECR_MINIWDL_ACCOUNT_ID=<some-value>
+export ECR_MINIWDL_REGION=<some-value>
+export ECR_MINIWDL_TAG=<some-value>
 ```
 
-These environment variables point to the ECR account, region, and tags of the Cromwell engine and WES adaptor containers respectively
+These environment variables point to the ECR account, region, and tags of the Cromwell ,Nextflow, and MINIWDL engine respectively
```
Comment (tneely) [inline README.md:114]: I don't think miniwdl has any casing. Can you change it from all caps?

Comment (evanyui) [reply README.md]: Fixed

File: README.md [hunk 5]
```diff
@@ -105,19 +106,25 @@ export ECR_CROMWELL_TAG=<some-value>
...
-export ECR_WES_ACCOUNT_ID=<some-value>
-export ECR_WES_REGION=<some-value>
-export ECR_WES_TAG=<some-value>
+export ECR_MINIWDL_ACCOUNT_ID=<some-value>
+export ECR_MINIWDL_REGION=<some-value>
+export ECR_MINIWDL_TAG=<some-value>
 ```
 
-These environment variables point to the ECR account, region, and tags of the Cromwell engine and WES adaptor containers respectively
+These environment variables point to the ECR account, region, and tags of the Cromwell ,Nextflow, and MiniWDL engine respectively
```
Comment (elliot-smith) [inline README.md:114]: NIT: Sorry for the last comment but can you please fix the spacing here?

**Review by waliag87 — APPROVED** [review]

**Review by waliag87 — APPROVED** [review]
